.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -2,7 +2,7 @@
 name: Epic
```

Comment:
We can move this to the planning repo as we plan to keep 1 issue: 1 pull request in this repo